Messon on EPEL8 is old version 0.49.
It has bug which prevents processing of split-lines.

diff -ru radare2-5.2.0/meson.build radare2-5.2.0.new/meson.build
--- radare2-5.2.0/meson.build	2021-04-11 18:09:56.000000000 +0200
+++ radare2-5.2.0.new/meson.build	2021-04-21 04:54:31.070632823 +0200
@@ -10,10 +10,7 @@
 r2_version_major = vers[0].to_int()
 r2_version_minor = vers[1].to_int()
 r2_version_patch = vers[2].split('-')[0].to_int()
-r2_version_number \
-  = (r2_version_major * 10000) \
-  + (r2_version_minor * 100) \
-  + (r2_version_patch)
+r2_version_number = (r2_version_major * 10000) + (r2_version_minor * 100) + (r2_version_patch)
 
 repo = '.'
 if meson.is_subproject()
